### PR TITLE
Add YAML Support

### LINKED ISSUE #7: Add support for Yaml definition files (in addition to json)

### PR DESCRIPTION
**Pull request by nebrass**
Add YAML files support
Closes #7

### CODE REVIEW

File: Source/AzPester.psm1 [hunk 1]
```diff
@@ -7,11 +7,38 @@ function Invoke-AzPester {
...
+        Write-Error 'The file type is not supported'
+        Exit
+    }
+
+    [String] $definitionJson = $null
+
+    if ($Definition -match '\.yaml$' -or $Definition -match '\.yml$') {
+        $rawYaml = Get-Content $Definition -Raw
+        # Convert YAML to PowerShell Object
+        $psYaml = (ConvertFrom-Yaml -Yaml $rawYaml)
```
Comment (jcorioland) [inline Source/AzPester.psm1:20]: where does this `ConvertFrom-Yaml` cmdlet comes from? Seems like I don't have it on my PowerShell installation.

Comment (nebrass) [reply Source/AzPester.psm1:20]: I forget to add it to the documentation. It's an external module: https://www.powershellgallery.com/packages/powershell-yaml/0.4.2

Comment (nebrass) [reply Source/AzPester.psm1:20]: Doc updated

**Review by jcorioland — APPROVED** [review]
Looks good to me.

**Pull request merged**